storage/bootstrap.go

```diff
@@ -89,18 +91,6 @@ func (bsm *bootstrapManager) cutoverTime(writeStart time.Time) time.Time {
 	return writeStart.Add(bufferFuture)
 }
 
-// flushTime determines the time we can flush data for after the bootstrap
-// process is finished (i.e., all the shards have bootstrapped).
-func (bsm *bootstrapManager) flushTime(writeStart time.Time) time.Time {
-	// NB(xichen): when bootstrap process is finished, we should have bootstrapped
-	// everything until writeStart + bufferFuture, which means the current time is
-	// at least writeStart + bufferFuture + bufferPast.
-	rops := bsm.opts.GetRetentionOptions()
-	bufferPast := rops.GetBufferPast()
-	bufferFuture := rops.GetBufferFuture()
-	return writeStart.Add(bufferFuture).Add(bufferPast)
-}
-
 // NB(xichen): Bootstrap must be called after the server starts accepting writes.
```

Comment:
Maybe we should enforce this btw?  Thoughts?  We can check if the db is open and if it is already return an error (I know that's not part of this change but just reading this comment makes me sad.